Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -143,6 +143,14 @@ class ProjectFileElements {
                              isReference: $0.isReference)
         })
 
+        target.copyFiles.forEach {
```

Comment:
Same here, I'd add a test to ensure that we generate file elements for the files referenced by the copy files.